libs/output-mapping/src/Writer/Helper/TypedColumnsHelper.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Writer\Helper;
+
+use Keboola\OutputMapping\Writer\Table\TableDefinition\TableDefinitionColumnFactory;
+use Keboola\StorageApi\Client;
+use Keboola\Utils\Sanitizer\ColumnNameSanitizer;
+
+class TypedColumnsHelper
+{
+    private static function getMissingColumns(array $tableInfo, array $config): array
+    {
+        $tableColumns = $tableInfo['columns'];
+        $configColumns = array_map(function ($columnName): string {
+            return ColumnNameSanitizer::sanitize($columnName);
+        }, array_keys($config['column_metadata']));
+
+        return array_diff($configColumns, $tableColumns);
+    }
+
+    public static function addMissingColumnsDecider(array $tableInfo, array $config): bool
```

Comment:
bylo by fajn tu mít nějaký víc zřejmější názvy proměnných, něco jako $currentTableInfo $newTableConfiguration nebo tak něco?

Reply:
upravil jsem to v obou helperech